pytket/extensions/quantinuum/backends/api_wrappers.py

```diff
@@ -513,6 +513,20 @@ def get_calendar(self, start_date: str, end_date: str) -> list[dict[str, str]]:
         jr: list[dict[str, str]] = res.json()
         return jr
 
+    def get_machine_list(self) -> list[dict[str, Any]]:
+        """returns the given list of the available machines
```

Comment:
```suggestion
        """Returns a list of the available machines
```
(not vital at all as a change, just the inconsistent lowercase from the original was bugging me)